docs/src/pages/guides/deploy.md

```diff
@@ -335,6 +335,18 @@ Use the following build settings:
 
 Then click the **Save and Deploy** button.
 
+## Render
+
+You can deploy your Astro project on [render](https://render.com/) following these steps:
+
+1. Create a [render.com account](https://dashboard.render.com/) and sign in
+2. Click the "**new+**"-button at the dashboard and select "**Static Site**"
+3. Connect your [GitHub](https://github.com/) or [GitLab](https://about.gitlab.com/) repository or alternatively enter the public URL of a public repository
+4. Give your website a name (1. to find it on the dashboard. 2. The onrender.com url is prefixed with it.), select the branch and specify the build command and publish directory  
```

Comment:
Minor suggestion (you can ignore): I don’t think you need to give 2 reasons in parentheses why you should name the project; I think it can be inferred.

Reply:
I think the point is more that you can just fully remove the blurb

Reply:
Do you mean removing everthing in point 4, except "build command" and "publish directory"?

Reply:
oh! i was looking at old code - what's there rn looks great!